src/sentry/static/sentry/app/components/modals/integrationDetailsModal.jsx

```diff
@@ -24,6 +27,18 @@ class IntegrationDetailsModal extends React.Component {
     this.props.onAddIntegration(integration);
   };
 
+  features(features) {
+    return features.map(feature => (
+      <StyledTag key={feature}>{feature.replace(/_/g, ' ')}</StyledTag>
+    ));
+  }
+
+  earlyAdopterLabel(provider) {
+    return EARLY_ADOPTER_INTEGRATIONS.some(key => key == provider.key) ? (
```

Comment:
You can use [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)